src/Controllers/FilesController.js

```diff
@@ -82,6 +82,19 @@ export class FilesController extends AdaptableController {
   expectedAdapterType() {
     return FilesAdapter;
   }
+
+
+    /**
+    * Stream video file by serving data in chunks if FilesAdapter is GridStoreAdapter.
+    * If not; handle the request as usual with "getFileData".
+    */
+  handleVideoStream(filename, range, res, contentType) {
```

Comment:
Maybe rename to `getFileRange(config, filename, range, res)` to be more like `getFileData()` or add a range attribute to `getFileData(config, filename, range, res)`